packages/salesforcedx-test-utils-vscode/src/orgUtils.ts

```diff
@@ -42,10 +42,10 @@ export const createScratchOrg = async (
   );
   const execution = new CliCommandExecutor(
     new SfdxCommandBuilder()
-      .withArg('force:org:create')
-      .withFlag('--definitionfile', `${scratchDefFilePath}`)
-      .withArg('--setdefaultusername')
-      .withJson()
+      .withArg('org:create:scratch')
+      .withFlag('--definition-file', `${scratchDefFilePath}`)
+      .withArg('--set-default')
+      .withJson(false)
```

Comment:
Why don't we want JSON anymore?

Reply:
we want it, the false is just to evaluate the addition of --loglevel